frontend/src/services/auth.ts

```diff
@@ -0,0 +1,57 @@
+import * as api from '../api';
+
+export interface User {
+  name: string;
+}
+
+const localStorageKey = 'token';
+
+class AuthService {
+  private _user: User | null = null;
+
+  get token() {
```

Comment:
should the getters in this class have return type?

Reply:
I'm not sure right now. I guess if it's good enough for the `strict` setting in `tsconfig` we can keep using the inferred return; we can always add them later if we see we prefer the explicit types.